edge-modules/api-proxy-module/docker/linux/arm64v8/Dockerfile

```diff
@@ -2,8 +2,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
-
-FROM arm64v8/nginx:1.19.7-alpine
```

Comment:
Just to be explicit, this is changin from Alpine to Ubuntu base image. Is that intentional?